website/src/pages/index.tsx

```diff
@@ -26,14 +26,14 @@ import Admonition from '@theme/Admonition';
 function EAP() {
   return (
     <div>
-      <Admonition type="info" title="Join our Early Adopter Program!">
+      <Admonition type="info" title="We are participating in Hacktoberfest 2022!">
         <p>
-          We are seeking users who are interested in helping us improve Podman Desktop. Register your interest to join
-          our Early Adopter Program by filling{' '}
-          <Link to="https://forms.gle/ow73dV7Ce3YLzoXH7">
-            <u>this form</u>
+          Do you love container tools and frontend technologies like Electron, Svelte and Tailwind CSS? Then come join Podman Desktop for Hacktoberfest 2022. You can start off with some issues by clicking{' '}
+          <Link to="https://bit.ly/3rrTCEP">
```

Comment:
not sure here why you're using  bit.ly instead of using `https://github.com/containers/podman-desktop/issues?q=is:open+is:issue+label:hacktoberfest` directly